commit 808696696837b8b8fc858f2e6f8d4e40e26e1308
Author: Mike Frysinger <vapier@gentoo.org>
Date:   Sat May 30 14:55:16 2015 -0400

    nscd: drop selinux/flask.h include
    
    Building nscd w/selinux enabled yields a warning which yields an error:
    In file included from selinux.c:32:0:
    /usr/include/selinux/flask.h:5:2: error:
     #warning "Please remove any #include's of this header in your source code."
    
    I've done just that and it builds cleanly with libselinux-2.4.

diff --git a/nscd/selinux.c b/nscd/selinux.c
index e433b1b..806763f 100644
--- a/nscd/selinux.c
+++ b/nscd/selinux.c
@@ -29,7 +29,6 @@
 #include <unistd.h>
 #include <sys/prctl.h>
 #include <selinux/avc.h>
-#include <selinux/flask.h>
 #include <selinux/selinux.h>
 #ifdef HAVE_LIBAUDIT
 # include <libaudit.h>
